packages/common/src/content/_internal/computeProps.ts

```diff
@@ -76,6 +89,7 @@ export function computeProps(
       extractFormatsList && extractFormatsList.split(",");
   }
 
+  // TODO: remove once .additionalResources is removed
```

Comment:
highlighting for visibility

Reply:
Thank you @abp6318 ! This is encapsulated in https://devtopia.esri.com/dc/hub/issues/10901